src/c/core/serialization/xrce_protocol_internal.h

```diff
@@ -201,7 +201,7 @@ typedef struct Property
 typedef struct PropertySeq
 {
     uint32_t size;
-    Property* data;
+    Property data[UXR_PROPERTY_SEQUENCE_MAX];
```

Comment:
Are we currently waste a `sizeof(Property)` size in the stack when a create message is created, right?

Reply:
Yes, it wastes only a `sizeof(char*)` more than the previous one. Do you prefer to remove it?

Reply:
It is ok for me, do not worry.